.npmrc

```diff
@@ -0,0 +1,3 @@
+strict-peer-dependencies = false
```

Comment:
Being [strict](https://pnpm.io/npmrc#strict-peer-dependencies) about peer dependencies sounds like a good thing. If there's a good reason to set this as false, I think we should at least document it.